crypto/ec/curve448/arch_32/arch_intrinsics.h

```diff
@@ -10,8 +10,8 @@
  * Originally written by Mike Hamburg
  */
 
-#ifndef HEADER_ARCH_32_ARCH_INTRINSICS_H
-# define HEADER_ARCH_32_ARCH_INTRINSICS_H
+#ifndef OPENSSL_ARCH_32_ARCH_INTRINSICS_H
```

Comment:
why the "arch" prefix?

Reply:
It's not from me, but I will change it to `OPENSSL_ARCH_INTRINSICS_H`.

Reply:
in that case, you should probably also look at arch_32/f_impl.h ?

Reply:
I‘ll take a global look and see what I‘ll find.

Reply:
The naming is actually consistent, it corresponds to the relative path inside the curve448 directory.

```grep
arch_32/arch_intrinsics.h:13:#ifndef OPENSSL_ARCH_32_ARCH_INTRINSICS_H
arch_32/arch_intrinsics.h:14:# define OPENSSL_ARCH_32_ARCH_INTRINSICS_H
arch_32/arch_intrinsics.h:27:#endif                          /* OPENSSL_ARCH_32_ARCH_INTRINSICS_H */
arch_32/f_impl.h:13:#ifndef OPENSSL_ARCH_32_F_IMPL_H
arch_32/f_impl.h:14:# define OPENSSL_ARCH_32_F_IMPL_H
arch_32/f_impl.h:60:#endif                  /* OPENSSL_ARCH_32_F_IMPL_H */
```

So I decided, that I'm not going to change it.